subprojects/integ-test/src/integTest/groovy/org/gradle/integtests/TaskDefinitionIntegrationTest.groovy

```diff
@@ -105,25 +102,24 @@ class TaskDefinitionIntegrationTest extends AbstractIntegrationSpec {
 
     def "can define tasks using task method expression"() {
         buildFile << """
-            ext.a = 'a' == 'b' ? null: task(withAction) << { }
+            ext.a = 'a' == 'b' ? null: task(withAction) { doLast {} }
```

Comment:
You could also do `task(withAction).doLast {}`. That is more in line with what happened before. I am also OK with leaving as is.

Reply:
I don't think it would make a difference thus I'll leave it as is in this case.